corehq/apps/export/static/export/js/models.js

```diff
@@ -923,7 +923,7 @@ hqDefine('export/js/models', [
      * @returns {Boolean} - True if it is the case name property False otherwise.
      */
     ExportItem.prototype.isCaseName = function () {
-        return this.path()[this.path().length - 1].name === 'name';
+        return this.path()[this.path().length - 1].name() === 'name';
```

Comment:
I guess they were. 
It is only used in two other places in the same template file and nested under some ifs. So it may have never come out in testing.

Reply:
@biyeun FYI in case this rings any bells? Looks like the other two places this is used (which are in this same file) you were the last person to touch

Reply:
this doesn't ring any bells